articles/what-is-self-supervised-learning/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/what-is-self-supervised-learning/
+title: What is Self-Supervised Learning?
+description: Self-supervised learning has recently been a hot topic in Artificial Intelligence. It is a form of supervised learning that doesn't require human input to perform data labeling. It can also be viewed as a branch of unsupervised learning. The results are obtained by models that analyze data, label and categorize information independently without any human input. In this article, we introduce self-supervised learning.
+author: willies-ogola
+date: 2021-03-23T00:00:00-18:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/what-is-self-supervised-learning/hero.jpg
+    alt: Self-Supervised Learning example image
+---
+Artificial Intelligence (AI) is a rapidly advancing field with various techniques being billed as the most promising to keep an eye on. One of those techniques is self-supervised learning.
+<!--more-->
+
+### Table of Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Introduction](#introduction)
+3. [What is Supervised Learning?](#what-is-supervised-learning)
+4. [Challenges with Supervised Learning](#challenges-with-supervised-learning)
+5. [Introduction to Self-Supervised Learning](#introduction-to-self-supervised-learning)
+6. [Applications of Self-Supervised Learning](#applications-of-self-supervised-learning)
+7. [Summary](#summary)
+7. [Wrapping Up](#wrapping-up)
+8. [References](#references)
+
+### Prerequisites
+
+Before reading this article, the reader needs to have little knowledge of Artificial Intelligence and Machine Learning. If you're still a beginner, feel free to read this [article](/section-engineering-education/supervised-learning-algorithms/) published on [Section](https://www.section.io/engineering-education/) to understand the differences between supervised learning or unsupervised learning on a larger scale. 
+
+### Introduction
+
+Artificial Intelligence (AI) is a rapidly advancing field with various techniques being billed as the most promising to keep an eye on. One of those techniques is self-supervised learning. You'll want to read this article to the end to learn more about it.
+
+### What is Supervised Learning?
+
+Supervised learning is a useful technique in deep learning. It is the most widely used type of learning when it comes to AI. It is a learning type where machines learn from labeled data to perform tasks such as predicting and classification of data. For example, a supervisor (someone who knows how to write answers) points out mistakes to a student during the learning process. The student ends up correcting the mistakes until he/she becomes good at them.
```

Comment:
Could you paraphrase the meaning of 'supervisor' in this context? Or add a few words, as it seems a bit ambiguous as is.

Reply:
I've edited and made it simpler to understand